mailpoet/lib/Analytics/UnsubscribeReporter.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
Please add `declare(strict_types = 1);` we want to add all new PHP files with the strict types enabled.